app/views/shared/_sidebar.html.erb

```diff
@@ -204,7 +204,6 @@
     <% path = display_course_path(lecture) %>
     <% active_class = get_class_for_path(path) %>
     <li class="sidebar-item nav-link <%= active_class %>
```

Comment:
At the end of the line, a `">` is missing, which prevents the link in the next line from being displayed properly.

Reply:
Oh, good catch